docs/admin/config/batch_changes.mdx

```diff
@@ -344,12 +344,37 @@ However, if the user deletion is permanent, deleting both account and data, then
 
 ## Batch Spec Library
 
-<Callout type="note"> Batch Spec Library is currently in Experimental.</Callout>
+The Batch Spec Library accelerates batch change adoption by offering curated examples and guided templates that make large-scale code modifications accessible to developers at every skill level.
 
-The Batch Spec Library is a collection of Batch Specs that can be used to create Batch Changes. Sourcegraph provides a few Batch Specs out of the box.
+The library distinguishes between [templates](#templates) and [library examples](#library-examples), where library examples are complete batch specs and templates are library examples with variables.
+
+Sourcegraph instances come with a couple of library examples out of the box. You can use the GraphQL APIs to [manage the Batch Spec Library](#managing-the-batch-spec-library).
+
+As a site admin, you can [feature records](#featured-records) to highlight the most useful examples for your organization.
+
+### Library examples
+
+Library examples are complete batch specs intended for advanced users who are comfortable working with YAML and writing code. These examples serve as inspiration and starting points for custom batch changes, providing reference implementations that can be adapted to specific use cases.
+
+Library examples are visible in the library pane when you are in the batch spec editor. They are not displayed by default in the templates list which is the entry point for users creating a batch change. See [choosing a template](/batch-changes/create-a-batch-change#choosing-a-template) for more details.
+
+### Templates
+
+<Callout type="note">Templates are supported in Sourcegraph v6.6 and more.</Callout>
+
+Templates are like library examples, but with variables for easy reuse across multiple batch changes. Templates provide the simplest path for users to create batch changes without needing to learn Batch Spec YAML syntax. Users select from a curated list of templates and complete a form with the required parameters, making batch change creation accessible to all team members regardless of their technical background.
+
+A library example becomes a template once it has at least one [variable](#variables) defined. It will then be displayed on the list of templates that a user sees when they click on [create a batch change](/batch-changes/create-a-batch-change).
+
+You can use variables to replace any text in the batch spec, allowing for flexible customization of repository queries, commands, commit messages, and more, but not for the name of the batch change.
```

Comment:
It seems we use "You" but also "the user" interchangeably. Do we want to pick one? It's a little confusing b/c the "you" is like we're speaking directly to the user, while using "the user" is more like a third person POV

Reply:
This part is a little awkward: "...but not for the name of the batch change." Maybe simply change to "You can use variables to replace any text in the batch spec except the batch change's name." I'm not sure the examples are necessary here.